mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/model/spi/MemberPropertyHandle.java

```diff
@@ -48,7 +52,7 @@ public Object get(Object thiz) {
 			if ( e instanceof InterruptedException ) {
 				Thread.currentThread().interrupt();
 			}
-			throw new SearchException( "Exception while invoking '" + member + "' on '" + thiz + "'" , e );
+			throw log.errorInvokingMember( member.getName(), thiz.toString(), e );
```

Comment:
Just pass `thiz` with type `Object` instead of `thiz.toString()` here, the JBoss logger should call `toString()` automatically, and if for some reason `thiz` is null, it will not throw an NPE like this code.

Reply:
Good idea. Shall I keep ` member.getName()` or I have to change this too?

Reply:
Right, I didn't see you switched from using `toString()` to using `getName`; we actually want `toString()` here, e.g. for a method we don't want just the name, but the return type too. Just pass "member" and expect JBoss Logging to use `toString()`.